design-documents/graph-ql/coverage/customer/Wishlist.graphqls

```diff
@@ -1,9 +1,12 @@
 type Mutation {
-    createWishlist(name: String!): ID # Multiple wishlists Commerce functionality
-    removeWishlist(id: ID!): Boolean # Commerce fucntionality - in Opens Source we assume customer always has one wishlist
+    createWishlist(name: String!, visibility: WishlistVisibilityEnum!): CreateWishlistOutput # Multiple wishlists Commerce functionality
+    deleteWishlist(wishlistId: ID!): Boolean # Commerce fucntionality - in Opens Source we assume customer always has one wishlist
```

Comment:
Propose to rename wishlistId to uid and change return type to an object that would contain list of the wishlists.